interface/wx/ribbon/bar.h

```diff
@@ -462,6 +462,57 @@ class wxRibbonBar : public wxRibbonControl
     */
     wxRibbonDisplayMode GetDisplayMode() const;
 
+    /**
+        Returns whether the bar contains multiple pages.
+
+        The default implementation always returns @c true.
+
+        @since 3.1.0
+    */
+    virtual bool HasMultiplePages() const;
+
+    /**
+        Returns @c true if the toggle button is currently moused over.
+
+        @since 3.1.0
+    */
+    bool IsToggleButtonHovered() const;
+
+    /**
+        Returns @c true if the help button is currently moused over.
+
+        @since 3.1.0
+    */
+    bool IsHelpButtonHovered() const;
```

Comment:
These functions are under "Implementation only." comment, I don't think they should be documented.

Reply:
OK, I removed this.